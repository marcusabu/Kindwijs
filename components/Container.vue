<template>
  <div class="page">
    <Header />
    <NavBar />
    <b-container fluid>
      <div class="logo-container">
        <Logo />
      </div>
      <b-container class="text-container">
        <slot></slot>
      </b-container>
    </b-container>
    <Footer />
  </div>
</template>

<script>
export default {}
</script>

<style scoped>
.page {
  display: flex;
  flex-direction: column;
  height: 100vh;
  position: relative;
}

@media (max-width: 1200px) {
  .logo-container {
    display: flex;
    justify-content: center;
  }
}

@media (min-width: 1200px) {
  .logo-container {
    position: absolute;
  }
}

.container-fluid {
  flex-grow: 1;
}

.text-container {
  padding-top: 2em;
  max-width: 55em;
}
</style>
