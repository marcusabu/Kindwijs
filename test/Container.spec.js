import Vue from 'vue'
import { shallowMount } from '@vue/test-utils'
import Container from '@/components/Container.vue'
import BootstrapVue from 'bootstrap-vue'

Vue.use(BootstrapVue)

describe('Container', () => {
  it('matches the snapshot', () => {
    const wrapper = shallowMount(Container, {
      stubs: {
        Logo: true,
        NavBar: true,
      },
    })
    expect(wrapper).toMatchSnapshot()
  })
})
