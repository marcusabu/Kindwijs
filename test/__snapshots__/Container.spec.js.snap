// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container matches the snapshot 1`] = `
VueWrapper {
  "_emitted": Object {},
  "_emittedByOrder": Array [],
  "isFunctionalComponent": undefined,
}
`;
